roscala/src/main/scala/coop/rchain/rosette/Ob.scala

```diff
@@ -99,9 +130,8 @@ trait Ob extends Base {
 
   def forwardingAddress: Ob = meta
 
-  def self: Ob = this
-
-  def inlineablePrimP: Prim = Prim.INVALID
+  // TODO:
+  //def inlineablePrimP: Prim = Prim.INVALID
```

Comment:
Sorry for that.
Mainly I put it in comments because this caused problems with my approach to define primitives.
Looking at the Rosette code again, I think we should do `def inlineablePrimP: Either[RblError, Prim] = Left(Invalid)` here.
I will change that.